tutorials/applications/QAOA and QAA to solve a MIS problem.ipynb

```diff
@@ -4,12 +4,12 @@
    "cell_type": "markdown",
```

Comment:
<p>I guess you can get rid of the generalization with <code>m</code> if you are going to stick to <code>m=1</code>.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>